### DISCUSSION TITLE
Oritech + Imer. Portal

### DISCUSSION THREAD

**Discussion by 1Lithium2** [iPortalTeam/ImmersivePortalsMod#1629] [Compatibility Issue Report]
### If it's a mod compatibility issue, please provide the link(s) to the incompatible mod(s)

Oritech Ui
https://www.curseforge.com/minecraft/mc
0.9.1
![Снимок экрана 2024-09-28 023536](https://github.com/user-attachments/assets/0ff564d7-6ca8-4e84-b0f1-3b5db2d911e4)
![Снимок экрана 2024-09-28 023610](https://github.com/user-attachments/assets/576bfaf6-e4e7-480e-98ab-6dde207fad59)

### Minecraft version

1.21.1

### Describe the issue

I know for sure that it's because of immersive portals.because I uninstalled all the mods, and installed one by one, and it was on this mod that the problem occurred.And I also put these 2 mods, without others, and the same problem.I describe what the problem is:The charge level and crafting progress is not displayed, but with the WTHIT mod, you can see in the tooltips above that everything works fine

**Comment by Nick56731** [comment]
please god fix this issue (also how did you get to see the power with WTHIT?)

**Comment by Kotane-TMKT** [comment]
Incompatibilities with Oritech still occur.
Oritech Mod links below.
https://modrinth.com/mod/oritech

The minimum environment is as follows. (I also write the version I use.)

--base
Minecraft 1.21.1
Fabric 0.16.10
--main mod
Immersive Portals | 6.0.6
Oritech | 0.13.2
--oritech dependencies
Fabric API | 0.114.0
GeckoLib | 4.7.3
Architectury API | 13.0.8
Althena | 4.0.1
oωo (owo-lib) |  0.12.10


Oritech machines don't work or behave strangely(almost don't work), animations and sounds don't play.
I haven't found any settings in the configuration to work around this.
Maybe an incompatibility with dependencies?

**Comment by Kotane-TMKT** [comment]
https://github.com/iPortalTeam/ImmersivePortalsMod/issues/1646#issue-2616149997

The author of Oritech identified this as an incompatibility with oωo and an issue with Immersive Portals.
